docs/source/executing-operations/demand-control.mdx

```diff
@@ -0,0 +1,554 @@
+
+## Calculating operation cost
+
+When your router receives a request, its query planner generates and sends a series of sub-requests to subgraphs. 
+
+To calculate the total cost of an operation, the router sums the total costs or weights based on sub-request's operation kind and the types of GraphQL elements of its fields.
+
+The cost of each operation kind:
+
+|           | Mutation | Query | Subscription |
+| --------- | -------- | ----- | ------------ |
+| kind      | 10       | 0     | 0            |
+
+
+The cost of each GraphQL element type, per operation kind:
+
+|           | Mutation | Query | Subscription |
+| --------- | -------- | ----- | ------------ |
+| Object    | 1        | 1     | 1            |
+| Interface | 1        | 1     | 1            |
+| Union     | 1        | 1     | 1            |
+| Scalar    | 0        | 0     | 0            |
+| Enum      | 0        | 0     | 0            |
+
+For example, assume the following query gets a response with six products and ten reviews:
+
+```graphql
+query ExampleQuery {
+  topProducts {
+    name
+    reviews {
+      author {
+        name
+      }
+    }
+  }
+}
+```
+
+Assuming each review having exactly one author, the total cost of the query is 26.
+
+<ExpansionPanel title="Example query's cost calculation">
+
+```text disableCopy=true showLineNumbers=false
+1 Query (0 cost) + 6 product objects (6) + 6 name scalars (0) + 10 review objects (10) + 10 author objects (10) + 10 name scalars (0) = 26 total cost
+```
+
+</ExpansionPanel>
+
+### Estimated and actual costs
+
+For an operation with list fields, the router must run the operation to get the actual number of items in its lists. Without actual list sizes, the cost of an operation can only be estimated before it's executed, where you assume the size of lists. 
+
+After an operation is executed, the actual cost per operation can be calculated with the actual list sizes.
+
+<Note>
+
+The difference between estimated and actual operation cost calculations is due only to the difference between assumed and actual sizes of list fields.
+
+</Note>
+
+### Measurement and enforcement modes
+
+When rolling out demand control, you first need to gather information about the queries that are already being executed against your graph so you can decide when to reject requests.
+
+The router's demand control supports a measurement mode that enables you to gather this information without impacting your running services. You can define telemetry instruments to monitor your operations and decide on their maximum cost threshold.  
```

Comment:
If `measurement` and `enforcement` aren't the actual mode names, italics can help distinguish/emphasize these.

```suggestion
The router's demand control supports a _measurement_ mode that enables you to gather this information without impacting your running services. You can define telemetry instruments to monitor your operations and decide on their maximum cost threshold.  
```